### PR TITLE
Lift Xext to xorg 1.4.2

### PR DESCRIPTION
**Pull request by uli42**

### CODE REVIEW

File: nx-X11/programs/Xserver/Xext/panoramiX.c [hunk 1]
```diff
@@ -129,11 +130,11 @@ static void XineramaValidateGC(GCPtr, unsigned long, DrawablePtr);
 static void XineramaChangeGC(GCPtr, unsigned long);
 static void XineramaCopyGC(GCPtr, unsigned long, GCPtr);
 static void XineramaDestroyGC(GCPtr);
-static void XineramaChangeClip(GCPtr, int, void *, int);
+static void XineramaChangeClip(GCPtr, int, pointer, int);
```
Comment (sunweaver) [inline nx-X11/programs/Xserver/Xext/panoramiX.c]: We don't want to re-introduce ``pointer`` here. Keep ``void *`` here.

File: nx-X11/programs/Xserver/Xext/panoramiX.c [hunk 2]
```diff
@@ -149,7 +150,7 @@ GCFuncs XineramaGCFuncs = {
 
 
 static Bool
-XineramaCloseScreen (ScreenPtr pScreen)
+XineramaCloseScreen (int i, ScreenPtr pScreen)
```
Comment (sunweaver) [inline nx-X11/programs/Xserver/Xext/panoramiX.c]: CloseScreen API is already more advanced. Drop ``int i`` here again and keep this line as is.

**Review by sunweaver — CHANGES_REQUESTED** [review]
Some minor changes requested. I presume that white-space changes originate in the used-as-template X.org files, so I have not commented on those (we leave them as part of this PR).

**Comment by uli42** [comment]
IIRC I have those fixed in the big 1.4.2 branch, but of course you are right

Am Fr., 9. Nov. 2018, 13:44 hat Mike Gabriel <notifications@github.com>
geschrieben:

File: nx-X11/programs/Xserver/Xext/panoramiX.c (hunk 1 above)
Comment (uli42) [reply nx-X11/programs/Xserver/Xext/panoramiX.c]: fixed

File: nx-X11/programs/Xserver/Xext/panoramiX.c (hunk 2 above)
Comment (uli42) [reply nx-X11/programs/Xserver/Xext/panoramiX.c]: done

**Review by sunweaver — APPROVED** [review]
LGTM!

**Pull request merged**